src/js/components/AppTaskStatsComponent.jsx

```diff
@@ -51,12 +52,24 @@ var AppTaskStatsComponent = React.createClass({
 
     let lifeTime = stats.lifeTime;
 
+    var averageSecondsHumanReadable =
+      Moment.duration(parseInt(lifeTime.averageSeconds), "seconds").humanize();
+
+    var medianSecondsHumanReadable =
+      Moment.duration(parseInt(lifeTime.medianSeconds), "seconds").humanize();
+
     return (
       <dl className="dl-horizontal dl-unstyled">
         <dt>Average</dt>
-        <dd>{lifeTime.averageSeconds} seconds</dd>
+        <dd>
+          {parseFloat(lifeTime.averageSeconds).toFixed(2)} seconds
```

Comment:
What happens is that because you're interleaving a string with a `{}` block, React will wrap any floating text nodes in a span so that it can assign it an ID and refer back to it later. If we wrap both inside a `{}` block then React needs to do nothing (no spans added).